examples/00_HH_simulator.ipynb

```diff
@@ -154,7 +169,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from HH_helper_functions import HHsimulator"
+    "from HH_helper_functions import HHsimulator\n",
+    "\n",
+    "# parameters (g_Na, g_K)\n",
+    "params = np.array([4.0, 1.5])\n",
+    "voltage_trace = HHsimulator(-70, params.reshape(1, -1), dt, t[:3000], I_inj[:3000])"
```

Comment:
Why is the [:3000] required? And why are we running it here, this also seems not to be used later on.

Reply:
it's not required it just runs shorter.

The idea to already run it here was to show that HH simulator returns a voltage trace because it is quite tucked away later and assumes quite some latent knowledge on HH using trace, activity etc. interchangeably 

but happy to remove it

Reply:
just removed it